src/attr.rs

```diff
@@ -174,6 +174,20 @@ impl Attribute {
         None
     }
 
+    /// Parses the tokens after the path as a [`Meta`](enum.Meta.html) if
+    /// possible.
+    #[cfg(feature = "parsing")]
+    pub fn parse_meta(&self) -> Result<Meta> {
+        use quote::ToTokens;
+
+        let mut tts = TokenStream::new();
+
+        self.path.to_tokens(&mut tts);
+        self.tts.to_tokens(&mut tts);
```

Comment:
This implementation is causing CI to fail because `ToTokens` requires the `printing` feature. I'm not sure how to solve this.